nms/app/state/AlarmsApiUtil.ts

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
_The diff does not display well for this file in Github. I just trust it is good._